packages/web/src/pages/profile-page/useProfileParams.ts

```diff
@@ -1,21 +1,17 @@
-import { cacheUsersSelectors } from '@audius/common/store'
-import { useSelector } from 'react-redux'
+import { useUserByParams } from '@audius/common/src/api/tan-query/useUserByParams'
 import { useLocation } from 'react-router-dom'
 
-import { AppState } from 'store/types'
 import { getPathname } from 'utils/route'
 import { parseUserRoute } from 'utils/route/userRouteParser'
 
-const { getUser } = cacheUsersSelectors
-
 export const useProfileParams = () => {
   const location = useLocation()
   const pathname = getPathname(location)
   const params = parseUserRoute(pathname)
-  const handle = params?.handle
-  const userId = params?.userId
+  const handle = params?.handle ?? undefined
+  const userId = params?.userId ?? undefined
+
+  const { data: user } = useUserByParams({ userId, handle })
```

Comment:
you might actually be able to do `useUserByParams(params)`
but this is fine too